chat-plugins/info.js

```diff
@@ -1897,10 +1898,12 @@ exports.commands = {
 	potd: function (target, room, user) {
 		if (!this.can('potd')) return false;
 
-		Config.potd = target;
-		Rooms.SimulatorProcess.eval('Config.potd = \'' + toId(target) + '\'');
+		let pkmn = Dex.getTemplate(target);
```

Comment:
``const``

Reply:
also why cant you just use `const template` instead of `pkmn`?

Reply:
I could @KrisOnline didn't think it would matter what a variable is called

Reply:
it matters if you want to remain consistent throughout PS's code in naming variables that fall along the lines of getting a Pokemon's template, such as any variable that is set to `Dex.getTemplate(target)` or `Object.assign({}, Dex.getTemplate(target))` or anything related

Reply:
Consistent variable naming helps a lot to make code more readable. You should definitely not be making any decision in code "just because you feel like it" or something.